### DISCUSSION TITLE
VulkanMod is the best optimization mod for macOS with Apple Silicon

### DISCUSSION THREAD

**Discussion by ghost** [xCollateral/VulkanMod#389] [General]
I'm on an iMac with the M3 chip.

It performs exceptionally well, surpassing Sodium and Vanilla by a wide margin. It seems as though this mod has been specifically optimized for Apple Silicon, allowing it to fully utilize the hardware capabilities.

While using Sodium, I typically experience frame rates ranging from 30 to 60fps with 32 chunks loaded, but it tends to overheat the M3 chip. On Vanilla, the game becomes increasingly laggy and glitchy as I move around.

However, with the Vulkan Mod, I can maintain a very high and stable frame rate with 32 chunks loaded, and my iMac remains cool throughout. Additionally, the chunks load significantly faster compared to Sodium or Vanilla.

I'm truly impressed by this mod and I hope that future updates will include support for features like mesh shading, frame generation, and shaders. Vulkan 1.3 would also be nice.

A big thank you to the creator and to all those who are contributing to this masterpiece.

**Comment by Enderteck** [comment]
This is expected as Apple doesn't support well Open GL Because they want to make it deprecated so that you use their Metal API. 

If you want good performances on Mac with openGL (without vulkanmod) you need to install Asahi Linux for example. Works great on Mac Silicon, better than MacOS especially for Minecraft as it supports modern OpenGL versions, you can also use Sodium and others to boost perofrmance over what Vulkanmod can do.

You can install it on only a part of your drive so that you can still boot into MacOS but have a better gaming experience with all windows steam games an stuff...

**Comment by LiamGamerp** [comment]
VulkanMod automatically uses MoltenVK on macOS, which translates Metal to Vulkan and Metal is the fastest graphics API on Apple Silicon as it has tight integration between software and hardware and hardware acceleration.
